### DISCUSSION TITLE
Onboarding Pull Request in Azure DevOps doesn't contain all dependency updates due to limitation of description field (3000 characters)

### DISCUSSION THREAD

**Discussion by pietergheysens** [renovatebot/renovate#17477] [Request Help]
Any ideas how to get the full overview of all detected dependency updates in the onboarding Azure DevOps PR? In the image below you can see that 33 pull requests are in scope but only 8 can be discovered in the description field.

![image](https://user-images.githubusercontent.com/1761094/187222213-0e889060-dc44-4800-bc43-30972dbbfb6e.png)

Is there a way to push the full dependency results in a plain text file during the renovate process and upload this somehow to a Wiki page or add the content to a file in the repository?

**Comment by rarkins** [comment]
There is no way to do that.

A quicker compromise may be to put the package file list and PR list into PR comments. Would you or a colleague be interested in contributing it?

**Reply by zivillian** [reply]
We ran into the same issue and are currently working on it.

Right now we have a working implementation that puts the package list and changelog into (up to two) separate PR comments when the description limit is reached. This should also work for other platforms with different limits and we haven even achieved 100% test coverage. It shouldn't be long before we can open a PR, but we're still testing our changes.

**Reply by darrexx** [reply]
We created #31616

**Reply by ghost** [reply]
As this is PR have been abandoned we ended up forking and applying a small patch to make it work for us. See: https://github.com/renovatebot/renovate/pull/31616#issuecomment-2747545980
